packages/app/package.json

```diff
@@ -30,6 +30,7 @@
     "@backstage/plugin-rollbar": "^0.2.9",
     "@backstage/plugin-scaffolder": "^0.4.2",
     "@backstage/plugin-sentry": "^0.3.4",
+    "@backstage/plugin-splunk-on-call": "^0.1.1",
```

Comment:
Again, would suggest we may not want to change this globally for the example app, but if needed adding it to the README for the plugin setup instructions?